BuyPolish/Pola/UI/ProductSearch/ScanCode/ScanCodeView.swift

```diff
@@ -86,5 +95,12 @@ final class ScanCodeView: UIView {
                 + (menuButton.bounds.height / 2)
                 - (logoButton.bounds.height / 2)
         )
+
+        NSLayoutConstraint.activate([
+            galleryButton.topAnchor.constraint(equalTo: flashButton.bottomAnchor, constant: scanCodeMargin),
```

Comment:
Proponuje taką kolejność guzików:

- klawiatura
- galeria
- latarka

Teraz w przypadku, gdy ktoś nie będzie miał latarki to będzie miał pustą dziurę między guzikami, a tak to po prostu zniknie ostatni przycisk.